### ISSUE TITLE
URL embedding and ADF database access

### ISSUE THREAD

**Issue by RiftsWRX** [naTmeg/ScriptedAmigaEmulator#13]
Hi there,

Love what you've done with this.  I was setting this up to host some of the old games I still have, and converted to ADFs, but wanted to know if there is a way of embedding the rom to load via the URL string itself.  Also, there are tons of preloaded demos and games in the UI.  Where are those stored and configured? I've tried grep'ing through the tree, looking for what file they are in, and can't seem to find them.

Thanks!
-Jorge

**Comment by RiftsWRX** [comment]
Let me update that last question.

I found where the DB is defined in index.js.  The next question though, is where the default location is for kick start?  If I manually plug in my Kickstart 1.3 rom, and a demo (like State of the Art), it works incredibly!  But, if I simply select state of the art, from the demo page, it just goes to a blank screen.  I'm assuming, it can't find the kickstart Rom.  I found the default paths for demos, aga demos, games, tools, etc.  But I guess I simply need to know where to properly house the kickstart, so it loads from the DB's main page.

Thoughts?  :)

-Jorge
